### PR TITLE
Black list

### PR DESCRIPTION
**Pull request by henryyang42**
Add black list feature.
To ban certain words, add it in `nthuoj/config/user_auth.cfg`

### CODE REVIEW

File: nthuoj/config/user_auth.cfg [hunk 1]
```diff
@@ -0,0 +1,5 @@
+[username]
+; add some phrase that should be banned below
+black_list = NTHU
```
Comment (nickelcheng) [inline nthuoj/config/user_auth.cfg]: I think we don't have to block NTUH because our project maybe renamed..

**Comment by nickelcheng** [comment]
+1 besides the comment. @bruce3557 please check this.

File: nthuoj/config/user_auth.cfg [hunk 2]
```diff
@@ -0,0 +1,4 @@
+[username]
+; add some phrase that should be banned below
+black_list = shit
```
Comment (henryyang42) [inline nthuoj/config/user_auth.cfg]: NTHU removed

**Comment by bruce3557** [comment]
+1, going to merge

**Pull request merged**